.env.development

```diff
@@ -92,6 +92,8 @@ DEFAULT_NUMBER_OF_RECORDS=10
 EMAIL_SUBJECT_PREFIX="[Simple DEVELOPMENT]"
 RAILS_MAX_THREADS=20
 SECRET_KEY_BASE=b90c991f1987f5be558af41e1fdc73741adbef2285f30a5bd500dcd7804b2ef1b49e8b07e03bd7154dcecd5050102f0f59cd182c8b40e48a66d416228dbd33fe
+USER_AUTH_LOCKOUT_IN_MINUTES=20
+USER_AUTH_MAX_FAILED_ATTEMPTS=5
```

Comment:
Normally I would want to define these in a single `.env` file and be done with it for all environments. I don't think there is a situation where these will vary, at least to start.

I notice we have our `.env` gitignored for this repo, though...is there a way to define these in one file that will apply everywhere?

Reply:
@harimohanraj89 or anyone - have any thoughts on this?

Reply:
Whoops, missed this during my last review, apologies. We don't have a global config file that's shared across environments. My go-to in this case would be to let these config vars be optional, and provide the "20" and "5" as defaults in the application layer. For example

```
class SomeModel
  LOCKOUT_PERIOD = Integer(ENV['USER_AUTH_LOCKOUT_IN_MINUTES'] || 20)
end
```

Reply:
PS If you go with the ["optional configuration" approach](https://github.com/simpledotorg/simple-server/pull/975/files#r428483384), we can remove these two from here.

Reply:
Got it, I'll definitely go with that. Easier to just set these in one spot for now.